### PR TITLE
HDDS-3701. Change Close container exception logging to debug in ContainerUtils#logAndReturnError

### PR DESCRIPTION
**Pull request by flirmnave**
## What changes were proposed in this pull request?

Change close container exception logging to debug level.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3701

## How was this patch tested?

CI

### CODE REVIEW

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java [hunk 1]
```diff
@@ -77,8 +78,13 @@ public static ContainerCommandResponseProto logAndReturnError(
       ContainerCommandRequestProto request) {
     String logInfo = "Operation: {} , Trace ID: {} , Message: {} , " +
         "Result: {} , StorageContainerException Occurred.";
-    log.info(logInfo, request.getCmdType(), request.getTraceID(),
-        ex.getMessage(), ex.getResult().getValueDescriptor().getName(), ex);
+    if (ex.getResult() == CLOSED_CONTAINER_IO) {
+      log.debug(logInfo, request.getCmdType(), request.getTraceID(),
+          ex.getMessage(), ex.getResult().getValueDescriptor().getName(), ex);
```
Comment (adoroszlai) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java:83]: Nit: Adding `isDebugEnabled()` check would let us avoid some unnecessary operations.

```suggestion
      if (log.isDebugEnabled()) {
        log.debug(logInfo, request.getCmdType(), request.getTraceID(),
            ex.getMessage(), ex.getResult().getValueDescriptor().getName(), ex);
      }
```

**Review by adoroszlai — COMMENTED** [review]
Thanks @flirmnave for taking care of this.

**Comment by flirmnave** [comment]
Thanks @adoroszlai for suggestions, I have update it.

**Comment by adoroszlai** [comment]
Thanks @flirmnave for updating the patch, LGTM.

**Review by ayushtkn — APPROVED** [review]
LGTM

File: hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java [hunk 2]
```diff
@@ -77,8 +78,15 @@ public static ContainerCommandResponseProto logAndReturnError(
       ContainerCommandRequestProto request) {
     String logInfo = "Operation: {} , Trace ID: {} , Message: {} , " +
         "Result: {} , StorageContainerException Occurred.";
-    log.info(logInfo, request.getCmdType(), request.getTraceID(),
-        ex.getMessage(), ex.getResult().getValueDescriptor().getName(), ex);
+    if (ex.getResult() == CLOSED_CONTAINER_IO) {
```
Comment (bshashikant) [inline hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java:81]: Its better to add debug for CONTAINER_NOT_OPEN as well.

Comment (flirmnave) [reply hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java]: Thanks @bshashikant for comment, I have update it.

**Comment by flirmnave** [comment]
Thanks for @ayushtkn for comment.

**Comment by flirmnave** [comment]
Thank @adoroszlai @ayushtkn @bshashikant for review and comment.

**Pull request merged**